### ISSUE TITLE
Update coding-standard to reflect that documentation should be sentence per line.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6537]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)| @tomprince reported|
|-|-|
|Trac ID|trac#6537|
|Type|enhancement|
|Created|2013-06-01 03:23:44Z|
|Branch|https://github.com/twisted/twisted/tree/sentence-per-line-6537|

This results in diffs that are easier to read, making documentation maintenance easier.

<details><summary>Searchable metadata</summary>

```
trac-id__6537 6537
type__enhancement enhancement
reporter__tom_prince tom.prince
priority__normal normal
milestone__ 
branch__branches_sentence_per_line_6537 branches/sentence-per-line-6537
branch_author__tomprince tomprince
status__closed closed
resolution__fixed fixed
component__core core
keywords__policy_documentation policy documentation
time__1370057024000000 1370057024000000
changetime__1386408948000000 1386408948000000
version__None None
owner__tom_prince tom.prince

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_1"></a>|@tomprince commented|
|-|-|

I wonder if this should also apply to docstrings and other multiline strings such as help text?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Let's try it out in howtos only for now and see how it goes.

For what it's worth, I'm more reluctant to apply this rule to any .py file where long lines are more likely to make life harder when actually writing and maintaining nearby software.  The howto xhtml files don't suffer this issue, since they're already not primarily about writing Python.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_4"></a>|@tomprince commented|
|-|-|

(In [39044]) Branching to sentence-per-line-6537.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_5"></a>|@tomprince commented|
|-|-|

I'm not sure how best to link to the blog (or whether that is even needed).

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set owner to @tomprince|
|-|-|

1.there's a typo in `formatted` on line 15
 1. same for `reformated` on line 29
 1. twisted will be around in 10 years but that blog(post) likely won't be.. I think you should either omit that url and include more of that 'fuller motivation' or leave that part out completely.

Thanks.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)|@tomprince removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra set owner to @tomprince|
|-|-|

Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_9"></a>|@tomprince set status to `closed`|
|-|-|

(In [40913]) Merge sentence-per-line-6537: Update coding-standard to reflect that documentation should be sentence per line.

Author: tom.prince
Reviewers: thijs
Fixes: [#6537](https://github.com/twisted/twisted/issues/6537)

This results in diffs that are easier to read, making documentation maintenance easier.
